Tasks/NodeToolV0/Tests/L0FirstDownloadSuccess.ts

```diff
@@ -9,6 +9,7 @@ let tmr: tmrm.TaskMockRunner = new tmrm.TaskMockRunner(taskPath);
 tmr.setInput('versionSource', 'spec');
 tmr.setInput('versionSpec', '11.3.0');
 tmr.setInput('checkLatest', 'false');
+tmr.setInput('nodejsMirror', 'https://nodejs.org/dist');
```

Comment:
@ismayilov-ismayil, 

The fix I have put in place to make the tests pass is to add this line:

```ts
tmr.setInput('nodejsMirror', 'https://nodejs.org/dist');
```

It's the same line I've added to all of the test files for NodeTool. 

However, this fix only makes sense if the following assumption is true: **The execution of the tests for NodeTool do not load the [default value](https://github.com/microsoft/azure-pipelines-tasks/pull/17500/files#diff-aada123c47bc7fec10b28b3b25e485fcdfe0800687619ce354d46a2793ed702aR81) for the inputs**. I believe this assumption is correct because this is on the line above my added line:

```ts
tmr.setInput('checkLatest', 'false');
```

And from the `master` branch you can see that `checkLatest` has a [default value](https://github.com/microsoft/azure-pipelines-tasks/blob/master/Tasks/NodeToolV0/task.json#L58) of `"false"`. 

If the tests for NodeTool were being primed with the default values defined in the `task.json` then my fix wouldn't be needed (nor would `checkLatest` need to be set to `false` in the tests).